src/application.rs

```diff
@@ -187,12 +187,23 @@ impl App {
             })
             .build();
 
+        // Switch to placeholder page
+        let action_placeholder = gio::ActionEntry::builder("placeholder")
+            .activate(|app: &Self, _, _| {
+                app.main_window()
+                    .imp()
+                    .stack
+                    .set_visible_child_name("placeholder");
```

Comment:
This isn't great, because it is accessing the internals of the window (a left-over from when I learned GTK, these should be private, but I haven't gotten around to it), please add a new window method.

Reply:
Done.